ivadomed/transforms.py

```diff
@@ -20,45 +20,45 @@
 
 def multichannel_capable(wrapped):
     @functools.wraps(wrapped)
-    def wrapper(self, sample, metadata):
+    def wrapper(self, sample, metadata, data_type):
```

Comment:
Oh.. what I meant in Slack was to add data_type in the metadata --> Is there advantages in doing the way you did? as it is only used for the resampling

Reply:
Changed it :)!